src/components/app/data/hooks/useBFF.test.jsx

```diff
@@ -129,90 +120,50 @@ const mockBFFDashboardData = {
 };
 
 describe('useBFF', () => {
-  const Wrapper = ({ children }) => (
+  const Wrapper = ({ routes = null, children }) => (
     <QueryClientProvider client={queryClient()}>
-      {children}
+      <MemoryRouter initialEntries={[routes]}>
+        {children}
+      </MemoryRouter>
     </QueryClientProvider>
   );
 
   beforeEach(() => {
     jest.clearAllMocks();
     useEnterpriseCustomer.mockReturnValue({ data: mockEnterpriseCustomer });
-    useEnterpriseFeatures.mockReturnValue({ data: { enterpriseLearnerBffEnabled: false } });
     fetchEnterpriseLearnerDashboard.mockResolvedValue(mockBFFDashboardData);
-    useLocation.mockReturnValue({ pathname: '/test-enterprise' });
     useParams.mockReturnValue({ enterpriseSlug: 'test-enterprise' });
```

Comment:
If we're rendering a legit router via `MemoryRouter` now, it feels a bit odd to continue mocking `useParams`. Should the rendering of the `MemoryRouter` itself make it possible to get access to the appropriate `useParams().enterpriseSlug` without mocking?

You likely need to render a `Route` component with the expected route path containing the route param `:enterpriseSlug` for the `useParams` to resolve correctly.

Something like:

```jsx
const Wrapper = ({ initialEntries = [], children }) => (
  <QueryClientProvider client={queryClient()}>
    <MemoryRouter initialEntries={initialEntries}>
      <Route path=":enterpriseSlug">
        {children}
      </Route>
    </MemoryRouter>
  </QueryClientProvider>
);
```